### ISSUE TITLE
Image Alt Attribute

### ISSUE THREAD

**Issue by jasonbutler42** [jasonbutler42/grillinfools#41]
All images should be checked for good alt text. This might be a good starting point: https://wordpress.org/plugins/seo-image/
